app/assets/javascripts/listings/components/listing/eligibility-section.html.slim

```diff
@@ -8,7 +8,7 @@ accordion-heading.lead
 
```

Comment:
not positive where the right spot to comment this is, but I noticed a difference between background colors of the eligibility section on the deployment and what's on [figma](https://www.figma.com/file/Xe5JvHHIIWK0Na0JOyoy4o/Habitat-listing?node-id=236%3A72). pairing on this sometime next week is probably a good idea 😅

Reply:
@jacksongonzales sorry I just finally took another look at this comment, and the figma color discrepancy is just an artifact from Yindi copying and pasting and editing I think. The color in this PR matches what's on prod